update/Pos/Update/Poll/Pure.hs

```diff
@@ -29,12 +29,13 @@ import           Pos.Update.Poll.Types     (BlockVersionState (..),
                                             psProposal)
 
 newtype PurePoll a = PurePoll
-    { getPurePoll :: NamedPureLogger (State Poll.PollState) a
-    } deriving (Functor, Applicative, Monad, CanLog, HasLoggerName)
+    { getPurePoll :: StateT Poll.PollState (NamedPureLogger Identity) a
+    } deriving (Functor, Applicative, Monad, CanLog, HasLoggerName, MonadState Poll.PollState)
 
 runPurePollWithLogger :: Poll.PollState -> PurePoll a -> (a, Poll.PollState, [LogEvent])
-runPurePollWithLogger ps =
-    (\((a, c), b) -> (a, b, c)) . flip runState ps . runNamedPureLog . getPurePoll
+runPurePollWithLogger ps pp =
+    let innerMonad = flip runStateT ps . getPurePoll $ pp
```

Comment:
In `universum` there's [`usingStateT`](https://github.com/serokell/universum/blob/3a4e3ac2f2f49cd88db878eef5e140df33536994/src/Monad/Trans.hs#L50) which is just `flip runStateT`. Very minor improvement but pattern of `flip runStateT` or `flip runReaderT` or `flip RunWriterT` occurs too often so there exist helper functions for that in `universum`. Maybe you don't know about such functions. But if you find it convenient to use them you can get rid of `flip` (if you don't like `flip` of course).

Reply:
I don't mind getting rid of `flip` here, and thanks for reminding me about those functions -- I saw that a while ago but eventually forgot about 'em and was too lazy to look them up 😁  

Allright, let me quickly amend the PR!